<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceWidget</class>
 <widget class="QWidget" name="FindReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Find:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="foundLabel">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>0 found</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findPreviousButton">
         <property name="text">
          <string>Find Previous</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findNextButton">
         <property name="text">
          <string>Find Next</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
